data/brands/amenity/restaurant.json

```diff
@@ -1012,11 +1012,20 @@
       "tags": {
         "amenity": "restaurant",
         "brand": "Din Tai Fung",
+        "brand": "鼎泰豐",
```

Comment:
In JSON, it’s invalid for an object to redefine the same property twice. There are some examples of `brand:en`, `brand:zh`, etc. in this repository.